### ISSUE TITLE
grunt running continuously taking up cpu

### ISSUE THREAD

**Issue by fransflippo** [pantsel/konga#575]
Running konga Docker image.

Grunt is running continously and taking up between 10% and 30% of my CPU. 

I'm expecting Konga to just sit idle if I'm not using it?

The logs also go crazy:
```
debug: KongProxyHooks:afterEntityList called() status
debug: KongProxyHooks:afterEntityFetch called() null
debug: Policy:dynamicNode `connection-id` is defined. 1
debug: Policy:dynamicNode `connection-id` is defined. 1
debug: KongProxyController:listAllEntityRecords:req.method GET
debug: KongProxyController:listAllEntityRecords:req.url /status
debug: KongProxyController:listAllEntityRecords:entity status
debug: KongService: listAllCb http://kong:8001/status?size=1000
debug: KongProxyController:req.method GET
debug: KongProxyController:req.url 
debug: KongProxyController:entity null
debug: Kong admin url => http://kong:8001
debug: KongProxyHooks:afterEntityList called() status
debug: KongProxyHooks:afterEntityFetch called() null
debug: Policy:dynamicNode `connection-id` is defined. 1
debug: Policy:dynamicNode `connection-id` is defined. 1
debug: KongProxyController:req.method GET
debug: KongProxyController:req.url 
debug: KongProxyController:entity null
debug: Kong admin url => http://kong:8001
debug: KongProxyController:listAllEntityRecords:req.method GET
debug: KongProxyController:listAllEntityRecords:req.url /status
debug: KongProxyController:listAllEntityRecords:entity status
debug: KongService: listAllCb http://kong:8001/status?size=1000
debug: KongProxyHooks:afterEntityFetch called() null
debug: KongProxyHooks:afterEntityList called() status
debug: Policy:dynamicNode `connection-id` is defined. 1
debug: Policy:dynamicNode `connection-id` is defined. 1
debug: KongProxyController:listAllEntityRecords:req.method GET
debug: KongProxyController:listAllEntityRecords:req.url /status
debug: KongProxyController:listAllEntityRecords:entity status
debug: KongService: listAllCb http://kong:8001/status?size=1000
debug: KongProxyController:req.method GET
debug: KongProxyController:req.url 
debug: KongProxyController:entity null
debug: Kong admin url => http://kong:8001
debug: KongProxyHooks:afterEntityList called() status
debug: KongProxyHooks:afterEntityFetch called() null
```
etc. etc.

Is this normal behavior?

**Comment by pantsel** [comment]
@fransflippo , yes when youre running in dev mode. There are file watchers monitoring everey file change, ready to recompile the /tmp folder. Set NODE_ENV=production to run in prod mode.

**Comment by fransflippo** [comment]
So I guess that would be addressed by #359 with the next Docker image release.
Thanks!
